mypy/stubutil.py

```diff
@@ -31,8 +39,9 @@ def parse_signature(sig):
     return (name, fixed, optional)
 
 
-def build_signature(fixed, optional):
-    args = fixed[:]
+def build_signature(fixed: MutableSequence[str],
+                    optional: MutableSequence[str]) -> str:
```

Comment:
Yes it can, though maybe I am missing something obvious here; I tried for a bit to have `args` be the only MutableSequence and have both `fixed` and `optional` not mutated.

I will make a commit as you suggested and keep trying for a bit to also unmutate `fixed`.